apps/client/src/login/types.tsx

```diff
@@ -0,0 +1,3 @@
+export interface LoginFormProps {
+  handleSubmit?: (event: React.FormEvent<HTMLFormElement>) => void;
+}
```

Comment:
Not blocking: Putting the props type in a separate file might hurt readability. Consider keeping this type colocated with the component.